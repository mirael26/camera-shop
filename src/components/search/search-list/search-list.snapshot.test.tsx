import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import { searchedProductsMock } from '../../../test/mocks';
import SearchList from './search-list';

test('SearchList component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<SearchList searchedProducts={searchedProductsMock} onLinkClick={() => ({})}/>);
  expect(asFragment()).toMatchSnapshot();
});
