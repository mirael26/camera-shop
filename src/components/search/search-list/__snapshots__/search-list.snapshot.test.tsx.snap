// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchList component renders correctly 1`] = `
<DocumentFragment>
  <ul
    class="form-search__select-list"
    data-testid="search-list"
  >
    <li
      class="form-search__select-item"
    >
      <a
        class="form-search__link"
        href="/catalog/product/4"
      >
        Орлёнок
      </a>
    </li>
    <li
      class="form-search__select-item"
    >
      <a
        class="form-search__link"
        href="/catalog/product/5"
      >
        Van Shot
      </a>
    </li>
    <li
      class="form-search__select-item"
    >
      <a
        class="form-search__link"
        href="/catalog/product/6"
      >
        Click Sap
      </a>
    </li>
  </ul>
</DocumentFragment>
`;
